### ISSUE TITLE
Dashboard Checkboxes unable to be changed, Reverting to Defaults

### ISSUE THREAD

**Issue by ACR-Jeff** [jellyfin/jellyfin-web#6294]
**Describe The Bug**
Hide watched content from 'Recently Added Media' checkbox doesn't allow unchecking, It continues to revert back to checked state after a page refresh or revisiting users Home preferences.

**Steps To Reproduce**
1. Go to Dashboard
2. Click on Users
3. Click on a Users profile
4. Click on Edit this user's profile, images and personal preferences.
5. Click on Home
6. Click on Hide watched content from 'Recently Added Media' check box to uncheck

**Expected Behavior**
Clicking on Hide watched content from 'Recently Added Media' check box to uncheck it should stay unchecked after a refresh.

**Logs**
No Errors in logs

**Screenshots**
![Image](https://github.com/user-attachments/assets/1adbbcce-9cde-4f3c-89d5-9038cf032a4e)

**System (please complete the following information):**
 - Platform: Debian GNU/Linux 12 (bookworm) x86_64
 - Browser: Firefox
 - Jellyfin Version: 10.10.1

**Additional Context**
I have cleared browser cache and Restarted server to see if the issue persists, It has persisted after.

**Comment by viown** [comment]
It seems to work fine on my end. Are you sure there aren't any errors in the JF logs?

**Comment by Dan-613** [comment]
It works fine for me also.

**Comment by ACR-Jeff** [comment]
Ok It's finely showing an error, If I click the checkbox to uncheck it and refresh the page it refreshes it back to checked automatically, Yet the logs don't appear to show anything related at the moment of the action being performed, The only thing it shows is requests and closed

[2024-11-04 15:43:09.695 -08:00] [ERR] An error occurred while accessing the plugin manifest: "https://repo.jellyfin.org/files/plugin/manifest.json"
System.Net.Http.HttpRequestException: Name or service not known (repo.jellyfin.org:443)
 ---> System.Net.Sockets.SocketException (0xFFFDFFFF): Name or service not known
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|285_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)
   at Jellyfin.Networking.HappyEyeballs.HttpClientExtension.AttemptConnection(AddressFamily addressFamily, SocketsHttpConnectionContext context, CancellationToken cancellationToken)
   at Jellyfin.Networking.HappyEyeballs.HttpClientExtension.OnConnect(SocketsHttpConnectionContext context, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(QueueItem queueItem)
   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.DecompressionHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at Microsoft.Extensions.Http.Logging.LoggingHttpMessageHandler.<SendCoreAsync>g__Core|5_0(HttpRequestMessage request, Boolean useAsync, CancellationToken cancellationToken)
   at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<SendCoreAsync>g__Core|5_0(HttpRequestMessage request, Boolean useAsync, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)
   at System.Net.Http.Json.HttpClientJsonExtensions.<FromJsonAsyncCore>g__Core|12_0[TValue,TJsonOptions](HttpClient client, Task`1 responseTask, Boolean usingResponseHeadersRead, CancellationTokenSource linkedCTS, Func`4 deserializeMethod, TJsonOptions jsonOptions, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Updates.InstallationManager.GetPackages(String manifestName, String manifest, Boolean filterIncompatible, CancellationToken cancellationToken)
[2024-11-04 15:43:09.710 -08:00] [INF] "Update Plugins" Completed after 0 minute(s) and 0 seconds
[2024-11-04 15:43:09.787 -08:00] [INF] Watching directory "/mnt/Movies"
[2024-11-04 15:43:10.231 -08:00] [INF] FFmpeg: "/usr/lib/jellyfin-ffmpeg/ffmpeg"
[2024-11-04 15:43:10.232 -08:00] [INF] ServerId: "1e186ee55b9546749cedf473f184dda7"
[2024-11-04 15:43:10.232 -08:00] [INF] Core startup complete
[2024-11-04 15:43:10.232 -08:00] [INF] Startup complete 0:00:09.4526703
[2024-11-04 16:12:47.646 -08:00] [INF] WS "192.168.1.169" request
[2024-11-04 16:13:42.214 -08:00] [INF] WS "192.168.1.169" closed
[2024-11-04 16:13:43.460 -08:00] [INF] WS "192.168.1.169" request
[2024-11-04 16:13:51.018 -08:00] [INF] WS "192.168.1.169" closed
[2024-11-04 16:13:52.135 -08:00] [INF] WS "192.168.1.169" request
[2024-11-04 16:16:56.898 -08:00] [INF] WS "192.168.1.169" closed
[2024-11-04 16:16:58.249 -08:00] [INF] WS "192.168.1.169" request

**Comment by Dan-613** [comment]
> An error occurred while accessing the plugin manifest: "https://repo.jellyfin.org/files/plugin/manifest.json"
> System.Net.Http.HttpRequestException: Name or service not known (repo.jellyfin.org:443)

Are you able to check that you have access that URL from the same network as your Jellyfin installation?
[https://repo.jellyfin.org/files/plugin/manifest.json](https://repo.jellyfin.org/files/plugin/manifest.json)

**Comment by ACR-Jeff** [comment]
Yep I am able to get access to the manifest via browser

![Image](https://github.com/user-attachments/assets/f208ea06-7e93-4117-b455-270dec38715d)


        "guid": "698b6f33-14ca-49b5-9d79-fc3c0ab941f5",
        "name": "Chapter Segments Provider",
        "description": "Create media segments based on chapters.\n",
        "overview": "Create media segments based on chapters.",
        "owner": "jellyfin",
        "category": "General",
        "versions": [
            {
                "version": "3.0.0.0",
                "changelog": "",
                "targetAbi": "10.10.0.0",
                "sourceUrl": "https://repo.jellyfin.org/files/plugin/chapter-segments-provider/chapter-segments-provider_3.0.0.0.zip",
                "checksum": "04c5162ab035f72fc52fb678aa75a9e0",
                "timestamp": "2024-10-26T18:20:41Z"
            }
        ]
    },
    {
        "guid": "d106ebe6-9ca8-4fbc-9cd1-a92a213da9f9",
        "name": "LrcLib",
        "description": "Download lyrics from https://lrclib.net/\n",
        "overview": "Download lyrics from https://lrclib.net/",
        "owner": "crobibero",
        "category": "Notifications",
        "versions": [
            {
                "version": "2.0.0.0",
                "changelog": "### Dependency updates ###\n- Update dependency Microsoft.Extensions.Http to 8.0.1 (#8) @renovate",
                "targetAbi": "10.10.0.0",
                "sourceUrl": "https://repo.jellyfin.org/files/plugin/lrclib/lrclib_2.0.0.0.zip",
                "checksum": "f329ab0dc589c0ac1aa82522e278d49e",
                "timestamp": "2024-10-29T00:27:05Z"
            }
        ]
    }
]

**Comment by Dan-613** [comment]
Also check you can access that URL from the Jellyfin machine itself. The error shows that the URL isnt accessible by Jellyfin. It might be that your Jellyfin server is unable to perform DNS lookups for example.

**Comment by ACR-Jeff** [comment]
Yes I am also able to access that URL from my jellyfin server machine.

**Comment by ACR-Jeff** [comment]
I've searched all previous log files via Jellyfin for that error, There were no other errors matching it, That was the only time that's visible of that particular error.

**Comment by Dan-613** [comment]
If that error hasn't appeared again, it may have just been z temporary Internet outage and unrelated to the issue you are having with unchecking that option.

Maybe use browser tools to look at the http request and response when you try to unchecked the option to see if anything obvious shows up there.

**Comment by viown** [comment]
Can you create a new user and see if it works there?

**Comment by ACR-Jeff** [comment]
I've created a new test user, Set it up normally, Went into dashboard and followed the same steps in my OP, Same issue happens. If I uncheck the box it will be unchecked until after a refresh of the page, The logs are showing requests and closed connections every time I refresh after unchecking the checkbox. So it doesn't appear any errors are shown for the particular issue. I checked out browser tools but didn't find anything yet that maybe useful, I will refresh my browser after writing this and try again.

[2024-11-05 10:00:38.591 -08:00] [INF] WS "192.168.1.169" request
[2024-11-05 10:02:39.110 -08:00] [WRN] Compiling a query which loads related collections for more than one collection navigation, either via 'Include' or through projection, but no 'QuerySplittingBehavior' has been configured. By default, Entity Framework will use 'QuerySplittingBehavior.SingleQuery', which can potentially result in slow query performance. See https://go.microsoft.com/fwlink/?linkid=2134277 for more information. To identify the query that's triggering this warning call 'ConfigureWarnings(w => w.Throw(RelationalEventId.MultipleCollectionIncludeWarning))'.
[2024-11-05 10:03:38.094 -08:00] [INF] WS "192.168.1.169" closed
[2024-11-05 10:03:39.077 -08:00] [INF] WS "192.168.1.169" request
[2024-11-05 10:03:43.147 -08:00] [INF] WS "192.168.1.169" closed
[2024-11-05 10:03:44.030 -08:00] [INF] WS "192.168.1.169" request
[2024-11-05 10:03:50.073 -08:00] [INF] WS "192.168.1.169" closed
[2024-11-05 10:03:50.993 -08:00] [INF] WS "192.168.1.169" request
[2024-11-05 10:03:55.138 -08:00] [INF] WS "192.168.1.169" closed
[2024-11-05 10:03:56.071 -08:00] [INF] WS "192.168.1.169" request

**Comment by ACR-Jeff** [comment]
I've noticed this doesn't only effect Hide watched content from 'Recently Added Media' Yet every checkbox within the dash board. I've even created test profiles. This happens with every browser I've tried, I've even tried a new install never used prior of chromium with default settings, Still no go. Every check mark is unable to be changed/unchecked, They continue going back to defaults. This is driving me to pull my hair out (What's left)
